### ISSUE TITLE
Glow ESP is not working.

### ISSUE THREAD

**Issue by M4jx** [Jire/Charlatano#137]
First of all I would like to thank for the update.
Now we get to the problem.

Since update 1.0 the OpenGL window doesn't appear in task bar and the Glow doesn't work (I don't know if the window has something to do with the esp [No console errors]). So I couldn't use it and stayed on 0.9.8. Now the same problem for the latest release.

Another question: Is it fine if I moved the new offsets to version 0.9.8 and continued using it?

Thanks.

**Comment by meister15** [comment]
Esp is not work..
control recoil- work
bunnyhop- work 
trigger- work
aim- work
flash- work

**Comment by kokolino1** [comment]
yeah aim and esp doesn't work for me neither, only rcs and bunnyhop since 1.0.0

**Comment by kominxe** [comment]
it's working but, somebody need small fix :p
